docs/references/experimental/msb/README.md

```diff
@@ -66,3 +66,51 @@ fossa experimental-link-user-defined-dependency-binary [DIR] \
 ```
 
 **Note**: Similar to the `analyze` subcommand, this supports the `--endpoint` and `--fossa-api-key` commands to customize how the client connects to FOSSA.
+
+### Troubleshooting
+
+#### Failing to resolve dependencies for FOSSA projects
+
+> _Note: This error may present during a `--detect-vendored` strategy._
+> _For more information on multi-stage builds see the [multi stage builds overview](../msb/README.md)._
```

Comment:
This link is pointing back to the document it's in. I think we need to delete or edit this blockquote

Reply:
whoops lmao, good catch